src/MacroExpander.js

```diff
@@ -200,6 +184,46 @@ class MacroExpander {
                 }
             }
         }
+
+        // Flow unable to figure out that this pathway is impossible.
+        // https://github.com/facebook/flow/issues/4808
+        throw new Error(); // eslint-disable-line no-unreachable
```

Comment:
I wonder if uglyifyjs would strip this out automatically?  It's probably easier to quiet `flow`'s warning about with a well-placed `$FlowFixMe` comment than it is to determine if this is getting into the output or not.

Reply:
It didn't. My concern with `$FlowFixMe` is that there's no single line within the method to put this on, and it would be put on the whole method, which means type safety would not be checked there. Would you still prefer I do it it? "throw new Error" already appears in the string, and I just tested and it makes no difference in the compressed size:

Sizes before gzip:
```
199950 katex1.min.js
199935 katex2.min.js
```
Sizes after gzip:
```
52621 katex1.min.js.gz
52621 katex2.min.js.gz
```

Reply:
It's too bad there's no good place to put the `$FlowFixMe`.  But since it doesn't make a difference when gzipped let's leave it in.  gzip is magic.